.github/workflows/build-test.yml

```diff
@@ -837,6 +848,6 @@ jobs:
           DISPLAY: 10
           VSC_JUPYTER_FORCE_LOGGING: 1
           VSC_JUPYTER_CI_TEST_VSC_CHANNEL: 'insiders'
-        uses: GabrielBB/xvfb-action@v1.7
+        uses: GabrielBB/xvfb-action@b706e4e27b14669b486812790492dc50ca16b465 # v1.7
```

Comment:
where do these values come from?

Reply:
or rather, how did you collect them to be inserted here?

Reply:
I did it manually by looking up the repositories corresponding to the actions and then finding suitable commit SHAs, e.g. https://github.com/GabrielBB/xvfb-action/releases -> https://github.com/GabrielBB/xvfb-action/commit/b706e4e27b14669b486812790492dc50ca16b465.

zizmor does not trust actions outside of `actions/*` by default and encourages pinning to specific commit SHAs instead of version tags.